<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_statistics</class>
 <widget class="QDialog" name="user_statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="age_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="weight_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Weight (lbs):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="height_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Height (cm):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="name_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>First Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="goals_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Fitness Goals:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gender_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Biological Gender:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="excercise_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Exercise Amounts:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="age_inp">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="placeholderText">
          <string>Example: 65</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="weight_inp">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="placeholderText">
          <string>Example: 140</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="height_inp">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="placeholderText">
          <string>Example: 183</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_inp">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="placeholderText">
          <string>Example: John</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="goals_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Maintain Weight</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weight Loss</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weight Gain</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="gender_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="exercise_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sedentary (little or no exercise)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lightly active (light exercise/sports 1-3 days/week)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Moderately active (moderate exercise/sports 3-5 days/week)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Very active (hard exercise/sports 6-7 days a week)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Super active (very hard exercise/sports, physical job, or training)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="submit_data_btn">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
